specs/mgt-search-suggestion.md

```diff
@@ -0,0 +1,161 @@
+# mgt-search-suggestion
+
+The suggestion component based on Graph Suggestion API, it provides a input box and flyout. When a query string typed in the input box, the flyout will rendered by the data from Graph Suggestion API, show some suggestions. it supports 3 kinds of entity types, File/Text/Suggestions. The component structure as below
```

Comment:
The support for 3 kinds of entities, is that based on a limitation from the API? Could it support custom entities?

Reply:
From the original design, 3 entity types were determined by our PM, so I will try extend it, make it support custom entities, thanks for it.

Reply:
What does a suggestions entity look like? And what about people entities?

Reply:
It current now supports all entity types once the entity type supported by the Graph Suggestion API.
There is a document for Suggestion API in 1P, but still ongoing for Graph endpoint.

https://docs.substrate.microsoft.net/docs/Substrate-APIs-and-services/Substrate-Search-Service-(3S)/3S-API-Reference/Suggestions.html?uid=suggestions